.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at:
+#
+#     https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+blank_issues_enabled: false
```

Comment:
I would keep this option true in case none of the template suits the user's needs